src/ngx_http_lua_common.h

```diff
@@ -207,6 +209,14 @@ struct ngx_http_lua_main_conf_s {
 };
 
 
+typedef struct ngx_http_lua_shm_zone_ctx_s {
+    ngx_log_t                   *log;
+    ngx_http_lua_main_conf_t    *lmcf;
+    ngx_cycle_t                 *cycle;
+    u_char                       data; /* ngx_shm_zone_t */
```

Comment:
I wonder why we couldn't just use the `ngx_shm_zone_t` type here if the data size is always fixed? Do I overlook anything?